packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -129,6 +137,9 @@ export const Article = ({
   licenseBox,
   modifier,
   messages,
+  messagebox,
+  messageForBox,
+  messageBoxLinks,
```

Comment:
Basert på dei andre kommentarene i fila.
```suggestion
  messageBoxLinks,
```